src/ontology/pato-edit.obo

```diff
@@ -22035,6 +22035,14 @@ xref: NLX:378
 is_a: PATO:0010006 ! cell morphology
 created_by: http://orcid.org/0000-0001-7258-9596
 
+[Term]
+id: PATO:0070027
+name: cell with bitufted dendrite morphology
```

Comment:
minor: this is named as is subclasses are cells. I think the implicit naming convention here dictates "bitufted dendrite cell morphology"